edb/lib/std/70-converters.edgeql

```diff
@@ -317,6 +317,16 @@ std::to_datetime(year: std::int64, month: std::int64, day: std::int64,
 };
 
 
+CREATE FUNCTION
+std::to_datetime(epochseconds: std::float64) -> std::datetime
```

Comment:
I think we also want to add a decimal variant of the function (Postgres seems to support that):

`std::to_datetime(epochseconds: std::decimal) -> std::datetime`

Reply:
Something interesting is happening.
I added a function to handle decimal:

```
CREATE FUNCTION
std::to_datetime(epochseconds: std::decimal) -> std::datetime
```

It works from the edgedb CLI:

```
SELECT <str>to_datetime(517795200.00n);
{'1986-05-30T00:00:00+00:00'}
```

But it fails when I run tests in Python:
```
edgedb.errors.QueryError: function to_datetime is not unique
```

It looks like it works in the CLI implemented with Rust, but fails with the Python code. I'll need to study how the Python code handles these overloads and if it's using Decimal class for decimal.

Reply:
Hm, Rust or no Rust doesn't make any difference as both interact with the DB via our binary protocol. Can you also copy/paste your Python test code here?

Reply:
And please push the latest code to this PR.

Reply:
I think this is due to implicit cast from int to float and decimal.  Adding an explicit int64 variant should help.

Reply:
Pushed: tests won't pass, but if you try with the CLI, you should see the result I pasted above.

Reply:
The Python test code is:

```py
    async def test_edgeql_functions_unix_to_datetime_03(self):
        dt = await self.con.fetchone(
            'SELECT <str>to_datetime(517795200);'
        )
        self.assertEqual('1986-05-30T00:00:00+00:00', dt)

    async def test_edgeql_functions_unix_to_datetime_04(self):
        dt = await self.con.fetchone(
            'SELECT <str>to_datetime(517795200.00n);'
        )
        self.assertEqual('1986-05-30T00:00:00+00:00', dt)
```

Reply:
Checking with the explicit `int64` version.

Reply:
> I think this is due to implicit cast from int to float and decimal. Adding an explicit int64 variant should help.

Yeah, but how do you explain repl vs unittest difference in behavior?

Reply:
`SELECT <str>to_datetime(517795200);` should fail in repl also.  Roberto seems to have been testing on a bigint literal, which would not trigger a lookup ambiguity.

Reply:
> SELECT <str>to_datetime(517795200); should fail in repl also. Roberto seems to have been testing on a bigint literal, which would not trigger a lookup ambiguity.

Good catch :)